docs/reference/nuget-config-file.md

```diff
@@ -496,6 +496,16 @@ Below is an example `nuget.config` file that illustrates a number of settings in
         <add key="MyRepo - ES" value="https://MyRepo/ES/nuget" />
     </packageSources>
 
+    <!-- Used to enforce that certain packages only come from specific package sources. -->
+    <packageSourceMapping>
+      <packageSource key="MyRepo - ES">
```

Comment:
We need to steer customers toward the best-practice of clearing these nodes in their repo config.

```suggestion
      <clear />
      <packageSource key="MyRepo - ES">
```